classes/image_mdev.oeclass

```diff
@@ -23,6 +24,33 @@ image_preprocess_mdev () {
 	rm -rf $cwd/${mdevdir}
 }
 
+# USE_image_mdev_nodes: space separated list of entries for mdev.conf
+# syntax:  <match>:<group>
+# example: spidev.*:spi
+CLASS_FLAGS += "image_mdev_nodes"
+DEFAULT_USE_image_mdev_nodes = ""
+do_rstage[postfuncs] += "do_rstage_mdev"
+def do_rstage_mdev(d):
+    import os
+
+    mdev_path = d.get("RSTAGE_DIR") + (d.get("mdevconf") or "/etc/mdev.conf")
+
+    if not os.path.exists(mdev_path):
+        return
+
+    mdev_file = open(mdev_path, 'a')
+    mdev_nodes = d.get("USE_image_mdev_nodes") or ""
+
+    if mdev_nodes:
+        mdev_file.write("\n# USE_image_mdev_nodes\n")
+
+    for mdev_node in mdev_nodes.split():
+        match,group = mdev_node.split(':')
+        mdev_line = "{:16s}0:{:8s} 660\n".format(match, group)
```

Comment:
Wouldn't it make sense to be able to control user and mode in the USE flag also?